itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -621,6 +607,72 @@ public void testFailoverDuringDumpWithPreviousFailed() throws Throwable {
     assertTrue(MetaStoreUtils.isTargetOfReplication(db));
     assertTrue(MetaStoreUtils.isDbBeingFailedOverAtEndpoint(db, MetaStoreUtils.FailoverEndpoint.TARGET));
     assertTrue(fs.exists(new Path(dumpPath, ReplAck.LOAD_ACKNOWLEDGEMENT.toString())));
+
+    primary.run("drop database if exists " + primaryDbName + " cascade");
+
+    WarehouseInstance.Tuple reverseDumpData = replica.run("use " + replicatedDbName)
+            .run("create table t3 (id int)")
+            .run("insert into t3 values (2)")
```

Comment:
insert different value as the last attempted one. e.g 10 or anything other than 2